dashboard/client/src/type/task.ts

```diff
@@ -39,6 +39,8 @@ export type Task = {
   end_time_ms: number | null;
   worker_id: string | null;
   profiling_data: ProfilingData;
+  error_type: string | null;
+  error_message: string | null;
```

Comment:
```suggestion
  error_message: string;
```

this is always there right?

Reply:
I think they should be null when no errors happen: https://github.com/ray-project/ray/blob/d21b8326103a2c65c9130c7d82c8a2a4bb4869ed/python/ray/experimental/state/common.py#L588-L591

And `error_message` also depends on `error_type`: if no error_type, there shouldn't be error_message.